charts/gateway-addons-helm/values.yaml

```diff
@@ -10,6 +11,38 @@ grafana:
   adminPassword: admin
   service:
     type: LoadBalancer
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: 'envoy-gateway'
+          orgId: 1
+          folder: 'envoy-gateway'
+          type: file
+          disableDeletion: false
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/envoy-gateway
+  dashboards:
+    envoy-gateway:
+      prometheus-stats:
+        gnetId: 1
+        revision: 1
+        datasource: Prometheus
+      # Scrape and load dashboards of Envoy Gateway
+      envoy-global-dashboard:
+        url: https://raw.githubusercontent.com/envoyproxy/gateway/main/charts/gateway-addons-helm/dashboards/envoy-global.json
```

Comment:
is this the best way to do it ? or should the dashboard we part of this chart ?

Reply:
the drawback of URL is, its not versioned

Reply:
got it, prefer loading dashboards from local disk

Reply:
change it to load dashboards from configmap